python/GafferUITest/PathListingWidgetTest.py

```diff
@@ -990,6 +996,9 @@ def testModelChangingData( self ) :
 				GafferUI.PathListingWidget.StandardColumn( "Value", "dict:value" )
 			],
 		)
+		# Sort by "Value" column as sorting by defaultNameColumn doesn't
+		# trigger a data update.
+		widget._qtWidget().header().setSortIndicator( 1, QtCore.Qt.AscendingOrder )
```

Comment:
I don't this test was intended to depend on sorting to trigger the initial update - I think that was just an oversight on my part. I think instead we should replace L1007-1010 with this, to be explicit about what we're querying :

```
		self.assertEqual( model.rowCount(), 0 )
		_GafferUI._pathModelWaitForPendingUpdates( GafferUI._qtAddress( model ) )
		self.assertEqual( model.rowCount(), 1 )
		self.assertEqual( model.data( model.index( 0, 1, QtCore.QModelIndex() ) ), None )
		_GafferUI._pathModelWaitForPendingUpdates( GafferUI._qtAddress( model ) )
		self.assertEqual( model.data( model.index( 0, 1, QtCore.QModelIndex() ) ), 10 )
```

Reply:
Fixed in a1b7bd3.